Homework/Manh/lesson-1/RangeSumQueryImmutable.java

```diff
@@ -0,0 +1,21 @@
+class NumArray {
+
+    int[] sums;
+    
+    public NumArray(int[] nums) {
+        this.sums = new int[nums.length];
+        for(int i = 0; i < nums.length; i++) {
+            sums[i] = i==0 ? nums[i] : nums[i] + sums[i-1];
```

Comment:
nitpick: in coding interview, should avoid ternary condition, it makes code hard to read.

Reply:
OK